src/draw/mod.typ

```diff
@@ -0,0 +1,4 @@
+#import "grouping.typ": intersections, group, anchor, copy-anchors, place-anchors, set-ctx, get-ctx, for-each-anchor, on-layer, place-marks
```

Comment:
What is draw/mod.typ for? We already have draw.typ.

Reply:
Its a rust-y way to do modules, its similar to `__init__.py` with python modules as well. I think moving it to `draw.typ` to avoid having to do a rename would work better